bigtable/admin.go

```diff
@@ -547,23 +547,60 @@ func (ac *AdminClient) TableInfo(ctx context.Context, table string) (*TableInfo,
 	return ti, nil
 }
 
-// SetGCPolicy specifies which cells in a column family should be garbage collected.
-// GC executes opportunistically in the background; table reads may return data
-// matching the GC policy.
-func (ac *AdminClient) SetGCPolicy(ctx context.Context, table, family string, policy GCPolicy) error {
+type gcPolicySettings struct {
+	ignoreWarnings bool
+}
+
+// GCPolicyOption is the interface to change GC policy settings
+type GCPolicyOption interface {
+	apply(s *gcPolicySettings)
+}
+
+type warnings struct{ ignore bool }
+
+func (w warnings) apply(s *gcPolicySettings) {
+	s.ignoreWarnings = w.ignore
+}
```

Comment:
nit: I think you can just define the type to be a bool, and have the `apply` function just set `s.ignoreWarnings` to be `true`. It's off by default and I don't think you want an option to set it to `false` when it is by default.

Reply:
Done